releasenotes/notes/Enable-conntrack-for-all-namespaces-5450b33d93de9dc8.yaml

```diff
@@ -0,0 +1,11 @@
+# Each section from every releasenote are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+fixes:
+  - |
+    Enable conntrack for all namespaces
```

Comment:
Can we change this to a more descriptive release note and specify what product it affects

Reply:
Done